### DISCUSSION TITLE
100% CPU usage for Conpot

### DISCUSSION THREAD

**Discussion by lewiswastaken** [telekom-security/tpotce#1460] [Q&A]
I'm very new to honeypots. I'm observing that Conpot is using 100% CPU as well as EWS (which I figure is normal) although Conpot doesn't seem to get any traffic at all and the logs look really empty. I'm viewing the logs in /data and I see some connections but not loads. I was just wondering what would the general cause for this be and should I be doing something? I checked the connections on the container and there didn't seem to be any either. Very confused!

Thanks (:

**Comment by t3chn0m4g3** [comment]
This is a known issue, but no final fix available. Have a look [here](https://github.com/mushorg/conpot/issues/564).
